share/jupyter/hub/templates/page.html

```diff
@@ -88,9 +89,21 @@ <h4 class="modal-title" id="{{key}}-label">{{title}}</h4>
 
     <span id="login_widget">
       {% if user %}
-        <a id="logout" class="btn navbar-btn btn-default pull-right" href="{{logout_url}}">Logout</a>
+        <a id="logout" class="navbar-btn btn-sm btn btn-default pull-right" href="{{logout_url}}"> <i class="fa fa-sign-out"></i> Logout</a>
+        <div class="dropdown navbar-btn btn-sm pull-right">
+          <button class="btn-sm btn btn-default dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-expanded="true">
+            <i class="fa fa-cogs"></i>
+            <span class="caret"></span>
+          </button>
+              <ul class="dropdown-menu" role="menu" >
```

Comment:
This dropdown shouldn't exist if the user isn't admin. I'm not sure it should ever exist, actually.